src/PrestaShopBundle/Resources/config/admin/services_alias.yml

```diff
@@ -0,0 +1,13 @@
+services:
```

Comment:
maybe name the file `services_deprecated` could be better?

Reply:
it's a good idea, but i think it's not necessary, since we can mark service as deprecated. :)